core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -610,20 +595,21 @@ bool core_mmu_find_table(vaddr_t va, unsigned max_level,
 	return ret;
 }
 
-bool core_mmu_shatter_superpage(struct core_mmu_table_info *tbl_info,
-				unsigned int idx, bool secure __maybe_unused)
+bool core_mmu_entry_to_finer_grained(struct core_mmu_table_info *tbl_info,
+				     unsigned int idx, bool secure __unused)
 {
 	uint64_t *new_table;
 	uint64_t *entry;
 	int i;
 	paddr_t pa;
 	uint64_t attr;
+	paddr_t block_size_on_next_lvl =
+		L1_XLAT_ADDRESS_SHIFT - (tbl_info->level) *
```

Comment:
The alignment here looks a bit funny, why isn't `L1_XLAT_ADDRESS_SHIFT` on the line above?
Also there's the `()` aren't needed.